alacritty_terminal/src/tty/unix.rs

```diff
@@ -148,7 +148,12 @@ fn default_shell_command(pw: &Passwd<'_>) -> Command {
 }
 
 /// Create a new TTY and return a handle to interact with it.
-pub fn new(config: &PtyConfig, window_size: WindowSize, window_id: Option<usize>) -> Result<Pty> {
+pub fn new(
+    config: &PtyConfig,
+    window_size: WindowSize,
+    alacritty_window_id: u64,
+    x11_window_id: Option<usize>,
```

Comment:
Aren't those the same? The window `id` x11 is using is basically winit window id?

Reply:
They are!